src/backend/gl/src/window/egl.rs

```diff
@@ -198,9 +208,8 @@ impl Inner {
 
         // Testing if context can be binded without surface
         // and creating dummy pbuffer surface if not.
-        let pbuffer = if egl.upcast::<egl::EGL1_5>().is_none()
-            && !display_extensions.contains("EGL_KHR_surfaceless_context")
-        {
+        // Some EGL 1.5 devices does not support EGL_KHR_surfaceless_context (Android Emulator, for example)
```

Comment:
are you sure? The extension is a [part of the 1.5 core](https://docs.imgtec.com/Architecture_Guides/Supported_Extensions_OpenGL_ES_EGL/topics/d0e1207.html):
> This functionality is core to EGL 1.5, so the extension is no longer needed.

Reply:
I'm worried about the opposite situation: we'd be running on an EGL-1.5 device that simply doesn't expose the extension (because it's in core).

Reply:
Nice catch, thank you! We need a runtime version check here.

Android Emulator running on EGL 1.4, but 1.5 are also available on the system.

Reply:
I'm a bit confused. So it happily gives you an 1.5 context even if you initialized 1.4?
That seems like a serious footgun. Perhaps, we could raise this with khronos-egl crate author in an issue?
Could you file it, please?

Reply:
> So it happily gives you an 1.5 context even if you initialized 1.4?

No, it gives me 1.4 context, but previous check `egl.upcast::<egl::EGL1_5>()` is true, because we have EGL 1.5 in Android 10+ systems.

Reply:
why are we not using 1.5 if it's available?

Reply:
`eglInitialize` return 1.4 version.